operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/EventSourceRetriever.java

```diff
@@ -19,10 +19,11 @@ default <R> ResourceEventSource<R, P> getResourceEventSourceFor(Class<R> depende
   <R> List<ResourceEventSource<R, P>> getResourceEventSourcesFor(Class<R> dependentType);
 
   /**
-   * Registers (and starts) the specified {@link EventSource} dynamically during the reconciliation.
-   * If an EventSource is already registered with the specified name, the registration will be
-   * ignored. It is the user's responsibility to handle the naming correctly, thus to not try to
-   * register different event source with same name that is already registered.
+   * When using this method is important to explicitly name the event source, since comparison is
```

Comment:
I'm not sure I understand this comment. This needs to be clarified.

Reply:
reformulated, does it makes more sense now? :)